src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -71,12 +71,8 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
     allPublishedPipeline(Get(rawlsAdminWorkspaces)) map {response =>
       response.entity.as[Seq[RawlsWorkspace]] match {
         case Right(srw) =>
-          logger.info("admin workspace list got: " + srw.length + " raw workspaces")
-          val published = srw.collect {
-            case rw:RawlsWorkspace if isPublished(rw) => rw
-          }
-          logger.info("admin workspace list collected: " + published.length + " published workspaces")
-          published
+          logger.info("admin workspace list reindexing: " + srw.length + " published workspaces")
```

Comment:
oooh, good catch. Updated in the second commit.

The Either is created directly from the `entity.as` call, using the underlying unmarshaller - it's all spray-json magic. Here's some doc on it: http://doc.akka.io/docs/akka/2.4.4/scala/http/routing-dsl/directives/marshalling-directives/entity.html